website/content/terminal/quickstart/faq.md

```diff
@@ -109,13 +109,44 @@ Do note that it is also possible that the menu or command is removed. If this is
 </p>
 </details>
 
+<details><summary>Why does my Portfolio file fail to load?</summary>
+<p>
+
+This will typically be the result of a formatting error. 
+
+- Check that all the necessary column titles are present.
+- Inspect the file to see if cells left blank have been filled unintentionally with 0 or NaN values.
+- A particular asset may not be able to load data. Check for valid historical data from the Stocks menu.
+- Format ticker symbols according to yFinance naming convention.
+- All dates must be entered as YYYY-MM-DD.
+- Transactions dated for today will fail to load historical data.
+- MacOS users should attempt to avoid using the Numbers application as it has a habit of changing the formatting while saving.
+      
+</p>
+</details>
+
 ## Source
 
 <details><summary>How do I update the OpenBB Terminal?</summary>
 <p>
 
-The terminal is constantly being updated with new features and bug fixes, hence, for your terminal to be update, you can use `git pull` to get the latest changes:
+The terminal is constantly being updated with new features and bug fixes. Updating will be different depending on the installation type:
```

Comment:
Please note this section resides under source which is referred to as the Python version from GitHub so if you wish to make this generic, add it to general.

Reply:
Gotcha, moved.

> Please note this section resides under source which is referred to as the Python version from GitHub so if you wish to make this generic, add it to general.